include/sys/fs/zfs.h

```diff
@@ -1029,6 +1032,8 @@ typedef struct vdev_stat_ex {
 	uint64_t vsx_agg_histo[ZIO_PRIORITY_NUM_QUEUEABLE]
 	    [VDEV_RQ_HISTO_BUCKETS];
 
+	uint64_t vsx_delays;
```

Comment:
This can be moved in to `vdev_stat_t` since it's a single uint64_t which will pack nicely in to the nvlist uint64 array passed to user space.